src/main/java/net/cytonic/cytosis/commands/moderation/ClearchatCommand.java

```diff
@@ -25,7 +25,7 @@ public ClearchatCommand() {
                 if (sender.hasPermission("cytosis.commands.clearchat")) {
                     for (Player online : Cytosis.getOnlinePlayers()) {
                         if (online.hasPermission("cytosis.commands.clearchat")) {
-                            online.sendMessage(MM."<GREEN>Chat has been cleared by \{player.getUsername()}!");
+                            online.sendMessage(MM."<GREEN>Chat has been cleared by \{Cytosis.getRankManager().getPlayerRank(player.getUuid()) + player.getUsername()}!");
```

Comment:
No. You need to use appending.
```java
online.sendMessage(MM."<GREEN>Chat has been cleared by".append(Cytosis.getRankManager().getPlayerRank(player.getUuid().getPrefix().append(player.getUsername())).append("!"));
```

Obviously, make sure it is syntactically correct and the styling applied properly.

Reply:
ok im about to commit the patched version